homework12/homework_12.py

```diff
@@ -0,0 +1,61 @@
+class Moving:
+    def __init__(self, name):
+        self.name = name
+
+    def move(self):
+        raise NotImplementedError('I do not understand this')
+
+
+class Animal(Moving):
+    def voice(self):
+        raise NotImplementedError('I do not understand this')
+
+
+class Transport(Moving):
+    def launch(self):
+        raise NotImplementedError('I do not understand this')
+
+
+class Duck(Animal):
+    def voice(self):
+        print('cria')
```

Comment:
Is this Russian duck? 🤔🤣

Reply:
Tak tochno major razvedki Marina Nikolaevna Utkina. pozivnoy Duck